src/SFA.DAS.Reservations.Functions.Reservations.AcceptanceTests/Steps/ReservationDeletedSteps.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using SFA.DAS.Reservations.Data;
+using SFA.DAS.Reservations.Domain.Accounts;
+using SFA.DAS.Reservations.Domain.Notifications;
+using SFA.DAS.Reservations.Domain.Reservations;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.Reservations.Functions.Reservations.AcceptanceTests.Steps
+{
+    [Binding]
+    public class ReservationDeletedSteps : StepsBase
+    {
+        public ReservationDeletedSteps(TestServiceProvider serviceProvider, TestData testData) : base(serviceProvider, testData)
+        {
+        }
+        
+        [Given(@"I have a reservation ready for deletion")]
+        public void GivenIHaveAReservationReadyForDeletion()
+        {
+            TestData.Reservation.Status = (short)ReservationStatus.Deleted;
+
+            var dbContext = Services.GetService<ReservationsDataContext>();
+
+            dbContext.Reservations.Add(TestData.Reservation);
+            dbContext.SaveChanges();
+        }
+        
+        [When(@"a delete reservation event is triggered")]
+        public void WhenADeleteReservationEventIsTriggered()
+        {
+            var accountsService = Services.GetService<IAccountsService>();
+            var mockAccountsService = Mock.Get(accountsService);
+
+            var userDetails = new UserDetails { CanReceiveNotifications = true, Email = "", Name = "", Role = "Owner", Status = 1, UserRef = "" };
+
+            mockAccountsService.Setup(x => x.GetAccountUsers(It.IsAny<long>())).ReturnsAsync(new List<UserDetails> { userDetails });
```

Comment:
Duplicated in the above file, can it be moved up?

Reply:
Done